magento/magento2
# Add dev:tests:run parameter to pass arguments to phpunit

### Description
A new option has been added to the `dev:tests:run` command:

```
  -c, --arguments=ARGUMENTS  Additional arguments for PHPUnit. Example: "-c'--filter=MyTest'" (no spaces) [default: ""]
```

### Fixed Issues (if relevant)

1. magento/magento2#9477: PHPUnit tests do not accept testsuite as argument

### Manual testing scenarios
1. Run `bin/magento dev:tests:run unit -c'--filter=DevTestsRunCommand`
2. Only the test for this PR should be executed

### Contribution checklist
